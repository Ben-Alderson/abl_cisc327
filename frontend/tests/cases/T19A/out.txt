error(You cannot transfer more than $1,000.00)


