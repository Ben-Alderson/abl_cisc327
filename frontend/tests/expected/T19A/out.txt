Enter command: Enter login: Enter command: Enter the account number to transfer from: Enter the account number to transfer to: Enter amount to transfer: ERROR: Can't transfer more than $1,000.00
Enter command: Enter command: 